src/components/userMenu/UserMenu.js

```diff
@@ -1,51 +1,115 @@
 import React, {useState} from "react";
-import Menu from "@material-ui/core/Menu";
 import MenuItem from "@material-ui/core/MenuItem";
 import {Redirect} from "react-router-dom";
 import {useLoggedIn} from "../../utils/hooks/useLoggedIn";
 import IconButton from "@material-ui/core/IconButton";
-import {Face} from "@material-ui/icons";
+import Avatar from "@material-ui/core/Avatar";
+import {jwtUtils} from "../../utils/jwt/jwtUtils";
+import Box from "@material-ui/core/Box";
+import MenuList from "@material-ui/core/MenuList";
+import ClickAwayListener from "@material-ui/core/ClickAwayListener";
+import Paper from "@material-ui/core/Paper";
+import Popover from "@material-ui/core/Popover";
+import {userTypes} from "../../utils/Enums";
 
 export default function UserMenu(props) {
     const [anchorEl, setAnchorEl] = useState(null);
     const [direction, setDirection] = useState(null);
+    const [open, setOpen] = useState(false);
     const {logout} = useLoggedIn()
 
+    const user = jwtUtils.getUser()
+
+    const stringToColor = (string) => {
```

Comment:
Nie możemy wziąć po prostu random color? Nie sądzę, żeby ktokolwiek nawet zwrócił uwagę na to, że ma jakiś wyszukany kolorek powstały z jego nazwiska, a tak powstałe kolory mogą się gryźć z naszą paletą kolorów. Można byłoby wybrać jakieś 10 kolorów, które będą git i je dodać do palety i z nich losować.

Reply:
Random color nie, bo to ma być avatar, czyli coś co identyfikuje usera, ale zgoda, że 10 dobrze dobranych kolorów byłoby lepsze